### ISSUE TITLE
Regression in generated models due to iCub3 modifictions

### ISSUE THREAD

**Issue by traversaro** [robotology/icub-model-generator#144]
After merging https://github.com/robotology/icub-model-generator/pull/143, in the committed models something strange happened, see https://github.com/robotology/icub-models/commit/cd4b2a6608cea70d0c455b68e18f414cb0820c0c . 
What is happening is that some new changes in the .ini files introduced by https://github.com/robotology/icub-model-generator/pull/138 has been removed. 
I think that what is happening is that the icub3 model has a copy of a lot of .ini files from the iCub2 model in https://github.com/robotology/icub-model-generator/tree/master/simmechanics/data/icub3/conf, and it installs them in https://github.com/robotology/icub-model-generator/blob/master/simmechanics/CMakeLists.txt#L334 in the same location of the same files of iCub2, overwriting in this way the files of iCub2 and all the recent modifications. I think the directory in https://github.com/robotology/icub-model-generator/tree/master/simmechanics/data/icub3/conf should contain only files actually used by iCub 3, and they should be installed in a directory different from the one of iCub2 .

**Comment by traversaro** [comment]
TL;DR: merging the iCub3 model broke again velocity control on iCub2 models.

**Comment by traversaro** [comment]
The regression is even more critical for limbs that are shared betwen iCub 2  and iCub 3, because in that case the iCub3 configuration files are overwriting the iCub 2 configuratin files.

**Comment by traversaro** [comment]
Better keep issues on icub-models for better visibility.
